### PR TITLE
Interactive list

### PR DESCRIPTION
**Pull request by sajid-munawar**
**In this milestone**
- Add a new javascript file(interactiveList.js)
-  Add event listener to the checkbox
- Update items object's value for completed key upon user actions.
- Implement a function for the "Clear all completed" button (use filter() method).
- Store the updated array of items in local storage.

### CODE REVIEW

File: src/interactiveList.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+        localData[id].completed = true;
+        localStorage.setItem('todos', JSON.stringify(localData));
+      } else if (!e.target.parentElement.classList.contains('checked')) {
+        localData[id].completed = false;
+        localStorage.setItem('todos', JSON.stringify(localData));
+      }
+    });
+  }
+});
+
```
Comment (zieeco) [inline src/interactiveList.js]: **[OPTIONAL]**

- Hey, your apps work fine however if a user checks the checkboxes and refreshes the page, the checked task disappears on the UI but their current state is saved in the localStorage. Please update your UI so it gets the tasks that were checked when the user refreshes the page to improve the user experience.

File: src/interactiveList.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+// export const checkboxs = document.querySelectorAll(".checkbox");
+
+// checkboxs.forEach((c) => {
+//   c.
+// });
```
Comment (zieeco) [inline src/interactiveList.js]: **[REQUIRED CHANGES]**

- Please, kindly remove unused code blocks to keep your codebase clean and maintainable.

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @sajid-munawar 


Great work with the ToDo List 👏 
You've done well implementing its features, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice design. ✔️
- Descriptive PR. ✔️
- Good commit message ✔️


## Required Changes ♻️
_Check the comments under the review._





## Optional suggestions
_Check the comments under the review._

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by OLIPLICHE — APPROVED** [review]
Hi @sajid-munawar , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**